src/Tribe/Admin/Notice/Install_Event_Tickets.php

```diff
@@ -0,0 +1,438 @@
+<?php
+/**
+ * Install_Event_Tickets notice.
+ * Install and/or activate Event Tickets when it is not active.
+ */
+
+namespace Tribe\Events\Admin\Notice;
+
+use WP_Upgrader;
+use WP_Ajax_Upgrader_Skin;
+use Plugin_Upgrader;
+use Tribe__Main;
+use Tribe__Admin__Notices;
+use Tribe__Template;
+
+/**
+
+ */
+class Install_Event_Tickets {
+
+	/**
+	 * Stores the plugin slug.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected static $plugin_slug = 'event-tickets';
+
+	/**
+	 * Stores the assets group ID for the notice.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected static $assets_group = 'tribe-events-admin-notice-install-event-tickets';
+
+	/**
+	 * Stores the instance of the notice template.
+	 *
+	 * @since TBD
+	 *
+	 * @var Tribe__Template
+	 */
+	protected $template;
+
+	/**
+	 * Register update notices.
+	 *
+	 * @since TBD
+	 */
+	public function hook() {
+		if ( ! is_admin() || ! current_user_can( 'install_plugins' ) ) {
+			return;
+		}
+
+		add_action( 'wp_ajax_nopriv_notice_install_event_ticketsr', [ $this, 'ajax_handle_notice_install_event_tickets' ] );
```

Comment:
Is the trailing "r" intentional?
`wp_ajax_nopriv_notice_install_event_ticketsr`

Reply:
Actually, do we need to trigger on `nopriv` Ajax calls (when the user is not logged in)?

Reply:
Looks like a typo 😬 And yeah, agree on the `nopriv` call. Seems that I added that initially by habit.